### DISCUSSION TITLE
How to force TorToiSe to use the same tone in dialogs?

### DISCUSSION THREAD

**Discussion by ut8uu** [neonbjb/tortoise-tts#626] [Q&A]
Hello!

I'm using the TorToiSe to create an audiobook of a friend of mine. It works pretty well in most cases, in both descriptions and dialogs. Unfortunately, sometimes dialogs are being created with too strange tones (for example, too high or screaming-like). Is this possible to somehow specify that the dialogs must be voiced using the same tone as the main text?

I met discussions where one proposed to add some context in the square parenthesis before the text, like [sad], [happy], or so. Unfortunately, it's not very convenient and is quite time-consuming. I do this in my spare time and for free, so some voice generalization would be a plus.

Thank you in advance!

**Comment by CoderCowMoo** [comment]
You can try using a language model (or even simpler classifier) to automatically describe a quote and then add it in each API call to the inference API. If the classifier allows for neutral then you can make it avoid adding the context.
